test/wwwtxt/test_page.rb

```diff
@@ -0,0 +1,37 @@
+require "minitest/autorun"
+require "./lib/wwwtxt/page"
+require "./lib/wwwtxt/plain_text_renderer"
+
+class TestPage < Minitest::Test
+
+  def constant_fetcher
+    Class.new do
+      const_set "FORMAT", "txt"
+
+      def self.page(url)
+        WwwTxt::Page.new(title, content)
+      end
+
+      def self.title
+        "My title"
+      end
+
+      def self.content
+        <<EOF
+        <p>body par 1</p>
+        <p>body par 2</p>
+EOF
```

Comment:
You might want to check this to make sure it's what you expect, I think they'll be extra whitespace you aren't expectig.